wp-content/themes/twentytwenty-child/js/scripts.js

```diff
@@ -0,0 +1,40 @@
+/**
+ * Business Card Tab Select (Helper Function)
+ *
+ * changes to page based off of chosen tab
+ * @param $ - jQuery selector
+ * @param page_num - page number to change to
+ */
+const bc_select_tab = ($, page_num) => {
+
+  $('#gform_target_page_number_4').val(page_num);
+  $('#gform_4').trigger('submit', [true]);
+
+}
+
+/**
+ * Business Card Tabs Function
+ *
+ * creates event listeners for each tab click
+ * @param $ - jQuery Selector
+ */
+const bc_tabs = ($) => {
+
+  // BC Tab 1: Your Information
+  $('#gf_step_4_1').live('click', () => bc_select_tab($, '1'));
```

Comment:
Does this prevent clicking tabs before the tab they're currently on is complete?

Reply:
Gravity Forms takes care of that on the backend, so form validation works the same on tabs as it does with **Back** and **Next** buttons.